src/ViewInstance.js

```diff
@@ -198,7 +198,7 @@ class ViewInstance extends React.Component {
     },
     checkCanBeOpenedInLinkedData: {
       type: 'okapi',
-      path: 'resource/check/:{id}/supported',
+      path: 'linked-data/inventory-instance/:{id}/import-supported',
```

Comment:
Does this enpoint require additional permissions?

Reply:
No. No changes are made on permissions for this URL.